.github/workflows/comment-on-pr.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         # here to be able to comment on the PR
 
       - name: Download benchmarks
-        uses: dawidd6/action-download-artifact@v2
+        uses: dawidd6/action-download-artifact@v3
```

Comment:
I don't think that this should have been changed. It seems there is no v3:
https://github.com/sympy/sympy/actions/runs/3328189773/jobs/5503897421
https://github.com/dawidd6/action-download-artifact

Reply:
I guess that's why there is no benchmarks report on this PR.

Reply:
Oh my. Sorry for messing this up. I'll open another PR with a fix.

Reply:
Sorry. My bad for not checking better.

Reply:
See #24181